src/dask_histogram/core.py

```diff
@@ -738,6 +742,11 @@ def to_delayed(self, optimize_graph: bool = True) -> list[Delayed]:
         return [Delayed(k, graph, layer=layer) for k in keys]
 
 
+def _hist_safe_sum(items):
```

Comment:
This is the duplicate; I didn't track which gets called.

Reply:
I think using the safe version may not be necessary for the sum.